tests/css_grid/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,79 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`grid.css 1`] = `
+div {
+  grid-template-areas:
+       "header header"
+       "main sidebar"
+       "footer footer";
+}
+
+div {
+  grid-template:
+        "a a a" 200px
+        "b b b" 200px
+        / 200px 200px auto;
+}
+
+div {
+  grid: [wide-start] "header header header" 200px [wide-end]
+      "footer footer footer" 25px
+      / auto 50px auto;
+}
+
+div {
+  grid-template-columns:
+      [full-start] minmax(1em, 1fr)
+      [main-start] minmax(40ch, 75ch)
+      [main-end] minmax(1em, 1fr)
```

Comment:
Can you add an example where it does the number normalization?

Reply:
Added!